test/js/node/http/node-http.test.ts

```diff
@@ -146,6 +146,29 @@ describe("node:http", () => {
             res.end("Path correct!\n");
             return;
           }
+          if (reqUrl.pathname === "/customWriteHead") {
+            function createWriteHead(prevWriteHead, listener) {
+              let fired = false;
+              return function writeHead() {
+                if (!fired) {
+                  fired = true;
+                  listener.call(this);
+                }
+                return prevWriteHead.apply(this, arguments);
+              };
+            }
+
+            function addPoweredBy() {
```

Comment:
This function is never called.

Reply:
Emm, I found that the locally generated esm files should also be included in the pr.